packages/gatsby/src/schema/resolvers.js

```diff
@@ -282,7 +283,23 @@ const getFieldNodeByNameInSelectionSet = (selectionSet, fieldName, info) =>
     return acc
   }, [])
 
+const defaultFieldResolver = (source, args, context, info) => {
```

Comment:
Not sure I like injecting this into args, very easy to get collision in that if someone actually has that in args. We can either wrap it to be a factory `(proxyArgs) => (source, args, context, info) =>` or maybe pass it through `info` object.

Reply:
yeah let's pass via `info` :+1: